finance-tracker/app/views/portfolio/index.html.erb

```diff
@@ -17,3 +17,8 @@
 </div>
 <div id="results"></div>
 
+<% unless @tracked_stocks.empty? %>
+  <%= render 'stocks/list' %>
+<% end %>
+
+
```

Comment:
?

Reply:
`tracked_stocks_empty?` returns true if no stocks are tracked. In this case, we don't want to render the list partial. So use `unless` to replace `if !tracked_stocks_empty?` to only render if there are stocks to list.

Reply:
I jz tested you whether you pay attention to the details not.
my `?` is on the **EXTRA** empty line you have in this file.